skimage/registration/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Image registration module.
+"""
```

Comment:
Looks good, thank you for staring on this! :) What do you think about including a more verbose description on what functions in this module might be used for and what "image registration" is?

Reply:
I think i have messed up something while commiting.. Should i make another branch and create a new pull request?

Reply:
That's not a problem. A good opportunity to learn some git. :muscle: 

I'm not sure how your local branch looks but the state of `mahamtariq58:docstring-to-skimage.registration` doesn't look that messed up. It's just a [whitespace change](https://github.com/scikit-image/scikit-image/pull/6791/files#diff-40a11602ab3926327a404fecb5c5b1170c47a113fcf42167092a359a1523851aR8) that I actually agree with. :smile: 

 You could just try to add the changes in https://github.com/scikit-image/scikit-image/pull/6791/commits/0a3d56e4df694def6ba35fbe937c6fc9de2cb3fb back to the module and create a new commit. When we merge, we usually squash all these different commits into a single commit.

In case your local branch doesn't reflect the remote state, you should be able to force the remote state with

```
git fetch --all
git reset --hard origin/docstring-to-skimage.registration
```

Let me know if that helps.